docs/source/markdown/podman-kill.1.md.in

```diff
@@ -24,33 +24,33 @@ Signal all running and paused containers.
 
 ## EXAMPLE
 
-Kill container with a given name
+Kill container with a given name:
 ```
 podman kill mywebserver
 ```
 
-Kill container with a given ID
+Kill container with a given ID:
 ```
 podman kill 860a4b23
 ```
 
-Terminate container by sending `TERM` signal
+Terminate container by sending `TERM` signal:
 ```
 podman kill --signal TERM 860a4b23
 ```
 
 Kill the latest container. (This option is not available with the remote Podman client, including Mac and Windows
-(excluding WSL2) machines)
+(excluding WSL2) machines):
 ```
 podman kill --latest
 ```
 
-Terminate all containers by sending `KILL` signal
+Terminate all containers by sending `KILL` signal:
 ```
 podman kill --signal KILL -a
 ```
 
-Kill container using ID specified in a given files
+Kill container using ID specified in a given files:
```

Comment:
Number mismatch. Should be **containers** (plural), **IDs**, **the** given files